packages/amplication-server/src/models/IBlock.ts

```diff
@@ -15,7 +16,8 @@ export abstract class IBlock {
   })
   createdAt!: Date;
 
-  resourceId?: string;
+  @IsString()
```

Comment:
This class is not used to get any input from users or API, so there is no reason to use validation decorators